cli/client.go

```diff
@@ -164,7 +164,7 @@ func (c *AppClient) ensureLoggedIn() error {
 	}
 
 	if c.conf.Auth == nil {
-		return errors.New("not logged in: run the following command to authenticate:\n\tviam auth")
+		return errors.New("not logged in: run the following command to login:\n\tviam login")
```

Comment:
Did I miss this change?

Are we swapping `auth` to `login`?

Reply:
Yep switched it [here](https://github.com/viamrobotics/rdk/commit/f3f67887a137428989f7f04e1328cd4d58203bdc#diff-3f48ce313a979a0ca7a9a3f674e1451b3381ea6d34c6ac31c68fb5313fcb9f34R80). `auth` is still a valid alias of `login`, though, so hopefully not backwards breaking.

Reply:
Makes sense! `login` is better